src/Microsoft.DotNet.Wpf/src/Shared/ref/AttributesForReferenceAssemblies.cs

```diff
@@ -3,8 +3,9 @@
 // See the LICENSE file in the project root for more information.
 
 //
+// This file specifies various assembly level attributes.
 //
 
 using System.Runtime.CompilerServices;
 
-[assembly:TypeForwardedTo(typeof(System.Windows.Input.ICommand))]
```

Comment:
this is weird, and maybe it's just a bug in GitHub, but did this `ICommand.cs` file get removed?

Reply:
I did remove it because it only contained a type forward.  I moved that type forward to the assembly attribute file so I could easily include it in the ref assembly.

I don't know why git decided it was renamed into my new assembly attribute file, but it did.